services/math.py

```
@@ -0,0 +1,281 @@
+from pyparsing import (
+    CaselessLiteral,
+    Combine,
+    Forward,
+    Literal,
+    Opt,
+    Word,
+    ZeroOrMore,
+    alphas,
+    nums,
+)
+from pydantic import BaseModel, Field
```

Flake8 (7.3.0)

[error] 'pydantic.Field' imported but unused

(F401)